src/command_modules/azure-cli-rdbms/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+0.2.4
```

Comment:
New version has been added yesterday. please update this.

Reply:
Done